node/src/service.rs

```diff
@@ -417,12 +424,11 @@ fn start_consensus(
 		relay_chain_slot_duration,
 		proposer,
 		collator_service,
-		// Very limited proposal time.
-		authoring_duration: Duration::from_millis(500),
-		collation_request_receiver: None,
+		authoring_duration: Duration::from_millis(1500),
```

Comment:
This doesn't mean execution time right? It's just how much time we have to propose a block